SU2_SOL/src/SU2_SOL.cpp

```diff
@@ -63,10 +63,11 @@ int main(int argc, char *argv[]) {
 
   /*--- Pointer to different structures that will be used throughout the entire code ---*/
 
-  COutput *output                = NULL;
-  CGeometry **geometry_container = NULL;
-  CSolver **solver_container     = NULL;
-  CConfig **config_container     = NULL;
+  COutput *output                 = NULL;
```

Comment:
Do we also need to update SU2_DOT, DEF, etc.?

Reply:
We could, if required. For now, only changes to SU2_SOL were needed to be able to reconstruct the solution of multi-instance problems.